src/widgets/barchart/bar.rs

```diff
@@ -155,7 +155,7 @@ impl<'a> Bar<'a> {
             };
 
             let width = value_label.len() as u16;
-            if width < max_width {
+            if width <= max_width {
```

Comment:
I think this is not going to do the right thing in all instances?

For example, can you add a test where one of the labels is `"🤦🏼‍♂️".into()`?

Reply:
I reverted the comparison back to `<` , but I am using width() instead